h2o-parsers/h2o-parquet-compat/h2o-parquet-v17-compat/testMultiNode.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Clean out any old sandbox, make a new one
+OUTDIR=sandbox
+rm -fr $OUTDIR; mkdir -p $OUTDIR
+
+# Check for os
+SEP=:
+case "`uname`" in
+    CYGWIN* )
+      SEP=";"
+      ;;
+esac
+
+function cleanup () {
```

Comment:
Unlink created files/whole folder ?

Reply:
I don't think we want that, we do want to keep any files that will help us diagnose any issues.

The purpose of this method is to kill the jvm processes.